src/planner/update_plan.cpp

```diff
@@ -118,7 +119,7 @@ void UpdatePlan::BuildInitialUpdatePlan(
 UpdatePlan::UpdatePlan(const parser::UpdateStatement *parse_tree)
     : update_primary_key_(false) {
   std::vector<oid_t> column_ids;
-  BuildInitialUpdatePlan(parse_tree, column_ids);
+  BuildInitialUpdatePlan(parse_tree, column_ids, nullptr);
```

Comment:
Why is this null? I would add a comment to explain what's going on and why you're not passing in `txn`.